manufacturing/README.md

```diff
@@ -1,13 +1,72 @@
-# Ventilator Build
+# Manufacturing design
 
-This contains details on the alpha and beta build assemblies, including parts information and assembly instructions.
+This section of the repository describes the physical design of the ventilator and the process of manufacturing the
+components and fully assembled units.
 
-* [Alpha Build Instructions](alpha-build-instructions)
-  * Note: Comprehensive step-by-step assembly instructions [can be found here](assets/covent-june-2020-assembly-instructions.pdf)
-* [Beta Build Plan](beta-build-instructions)
-* [Production methods for custom components](assets/covent-june-2020-production-methods-for-custom-components.pdf)
+Broadly there are two types of ventilator builds we are currently testing with:
+- open table-top prototypes for testing components and various pneumatic configurations, kindly referred to as "pizza build"
+- "enclosed build", more like the final product; more expensive to make and slower to iterate on
 
-# Part Purchasing Guidelines
+| Pizza build  |  Enclosed build   |
+|:---------------------:|:--------------------:|
+![](pizza_build/graphics/3_pizzas_small.jpg) |![](enclosed_build/graphics/india1.jpeg) |
```

Comment:
Heads up - for some reason I'm not seeing images in previous within this table.

Reply:
If you had pulled locally, you probably don't have LFS enabled. It's a new thing we have for binary files. Check repo wiki for updates.